client/src/js/dd_main.js

```diff
@@ -89,25 +96,25 @@
         intervalMillis: 1
     };
     var cruise = new vis.VisibilityController(document.getElementById('cruise-wrapper'), copts);
-    speedDial.value(100);
+    speedDial.value(120);
     batteryDial.value(100);
     solarReadout.value(0);
     motorReadout.value(0);
     var date = new Date();
-    document.getElementById('status').innerHTML = date.toLocaleTimeString();
+    document.getElementById('status').innerHTML =
```

Comment:
This seems to be the same as the function in `setInterval`. Can we have them share code?